### PR TITLE
Make legacy functions crash on osx core contexts

### PR DESCRIPTION
**Pull request by fkaa**
Default behaviour is to just ignore the functions.

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -36,6 +36,9 @@ cocoa = "0"
 core-foundation = "0"
 core-graphics = "0"
 
+[target.x86_64-apple-darwin.dependencies.cgl]
+git = "https://github.com/servo/cgl-rs/"
```
Comment (tomaka) [inline Cargo.toml]: Dependencies should come from crates.io

Comment (fkaa) [inline Cargo.toml]: Completely forgot about that, fixed

**Comment by tomaka** [comment]
And needs a rebase!

**Comment by tomaka** [comment]
Thanks

**Pull request merged**